scripts/vet.sh

```diff
@@ -178,7 +178,7 @@ done
 
 # Collection of revive linter analysis checks
 REV_OUT="$(mktemp)"
-revive -set_exit_status=1 -exclude "reflection/test/grpc_testing_not_regenerate/" -formatter plain ./... >"${REV_OUT}" || true
+revive -set_exit_status=1 -exclude "reflection/test/grpc_testing_not_regenerate/" -formatter plain -config "$(dirname "$0")/revive.toml" ./... >"${REV_OUT}" || true
```

Comment:
I think we can safely combine lines 180-184 to be something like:

```suggestion
revive \
  -set_exit_status=1 \
  -exclude "reflection/test/grpc_testing_not_regenerate/" \
  -exclude "**/*.pb.go" \
  -config "$(dirname "$0")/revive.toml" \
  ./...
```

Reply:
Done